drfpasswordless/serializers.py

```diff
@@ -218,40 +218,40 @@ def validate(self, attrs, user=None):
             callback_token = attrs.get('token', None)
             user = user or User.objects.filter(**{alias_attribute_name: alias}).first()
 
-            token = CallbackToken.objects.get(**{'user': user,
+            tokens = CallbackToken.objects.filter(**{'user': user,
                                                  'key': callback_token,
                                                  'type': CallbackToken.TOKEN_TYPE_AUTH,
                                                  'to_alias_type': alias_type.upper(),
-                                                 'to_alias': attrs.get('mobile', None) or attrs.get('email', None),
-                                                 'is_active': True})
+                                                 'to_alias': attrs.get('mobile', None) or attrs.get('email', None),})
 
-            if token.user == user:
-                # Check the token type for our uni-auth method.
-                # authenticates and checks the expiry of the callback token.
-                if not user.is_active:
-                    msg = _('User account is disabled.')
-                    raise serializers.ValidationError(msg)
+            if not tokens.count():
```

Comment:
very rare situation, and users don't have many tokens in the first place so let's minimize db calls and fetch all then filter in Python